homeassistant/components/onewire/onewireentity.py

```diff
@@ -0,0 +1,105 @@
+"""Support for 1-Wire entities."""
```

Comment:
Please rename this module and separate words with underscore.

Reply:
Happy to rename the module.
But if I rename the class, then I have to tweak the [OWFS implementation](https://github.com/home-assistant/core/pull/42445/files#diff-d05b939af7a589516b66841111eae4ca7f2bdb2f3f3635d12cdeda77f0f95073L397) `class OneWireOWFS(OneWire):`

Reply:
And I was trying to avoid making any changes to the deprecated OWFS implementation (hard to tip-toe around it!!)

Reply:
Why? `onewireentity.py` is a new module. It should be named eg `onewire_entity.py`.

Reply:
I had misunderstood - I thought you were talking about the classes inside.
Should be good in 26007e7